modules/website/beaconApp/src/app/app.component.ts

```diff
@@ -12,23 +13,23 @@ export class AppComponent {
   isAuthenticated: boolean;
   login: boolean;
 
-  constructor(public oktaAuth: OktaAuthService, private appConfigService: AppConfigService, private router: Router) {
-     this.oktaAuth.$authenticationState.subscribe(
-       (isAuthenticated: boolean)  => this.isAuthenticated = isAuthenticated
+  constructor(public oktaAuth: OktaAuthStateService, private appConfigService: AppConfigService, private router: Router) {
+     this.oktaAuth.authState$.subscribe(
+       (value: AuthState)  => this.isAuthenticated = value.isAuthenticated
      );
    }
 
    ngOnInit() {
      this.login = this.appConfigService.login;
-     this.oktaAuth.isAuthenticated().then((auth) => {
-       this.isAuthenticated = auth;
-       if(this.login == true && this.isAuthenticated == false ){
-         this.router.navigate(['/login']);
-       }
-     });
+     //this.oktaAuth.isAuthenticated().then((auth) => {
```

Comment:
same here: this was more just me disabling the existing thing - you'll have to make it match how the new one approaches this